util/src/main/java/tech/pegasys/teku/util/config/TekuConfiguration.java

```diff
@@ -67,6 +67,7 @@
   // TODO (#1918): The following two options will eventually be moved to the validator subcommand
```

Comment:
I'd remove this comment while you're passing through here (it would be the next 6 options that moved anyway and the TODO is just not useful).